slidez-core/build.gradle

```diff
@@ -1,37 +1,45 @@
 plugins {
-	id 'org.springframework.boot' version '2.5.2'
-	id 'io.spring.dependency-management' version '1.0.11.RELEASE'
-	id 'java'
+    id 'org.springframework.boot' version '2.5.2'
+    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+    id 'java'
+    id "io.spring.javaformat" version "0.0.27"
+}
+apply plugin: 'io.spring.javaformat'
+apply plugin: 'checkstyle'
+
+checkstyle {
+    toolVersion = "8.29"
+    configFile = rootProject.file('src/main/java/com/binarystudio/academy/slidez/infrastructure/checkstyle/checkstyle.xml')
```

Comment:
You shouldn't put linter files into the source code. Please, move it to java project root OR create a new folder `conofig` and place it there

Reply:
I moved it, please, check